.vscode/launch.json

```diff
@@ -0,0 +1,34 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+
+    {
+      "type": "node",
+      "name": "vscode-jest-tests.v2.fizzbuzz_javascript",
+      "request": "launch",
+      "args": [
+        "--runInBand",
+        "--watchAll=false",
+        "--testNamePattern",
+        "${jest.testNamePattern}",
+        "--runTestsByPath",
+        "${jest.testFile}"
+      ],
+      "cwd": "${workspaceFolder}",
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen",
+      "disableOptimisticBPs": true,
```

Comment:
For some reason this option gives me a warning in the `Problems` panel, It looks like other folks sometimes get this warning even though things work as expected:

> Property disableOptimisticBPs is not allowed.

I understand that it's used for Babel and TypeScript to ensure that mappings are complete before placing breakpoints so they are placed correctly and debugging works as expected, but I'm having a hard time finding documentation on this option to understand why it's throwing a warning.

Reply:
I also tried debugging with it removed this morning and had no issue 👍